### ISSUE TITLE
Unable to start velocity vanish

### ISSUE THREAD

**Issue by Frerduro** [Syrent/VelocityVanish#252]
### Server backend version

git-Purpur-1985 (MC: 1.19.4)*

### Server backend Java version

openjdk version "17.0.8.1" 2023-08-24 OpenJDK Runtime Environment Temurin-17.0.8.1+1 (build 17.0.8.1+1) OpenJDK 64-Bit Server VM Temurin-17.0.8.1+1 (build 17.0.8.1+1, mixed mode, sharing)

### Server Velocity version

Velocity 3.2.0-SNAPSHOT (git-19abb909-b265)

### Server Velocity Java version

openjdk version "17.0.8.1" 2023-08-24 OpenJDK Runtime Environment Temurin-17.0.8.1+1 (build 17.0.8.1+1) OpenJDK 64-Bit Server VM Temurin-17.0.8.1+1 (build 17.0.8.1+1, mixed mode, sharing)

### VelocityVanish version

3.26.0-build.24

### Describe the bug

Velocity fails to load on backend servers

### Expected behavior

Velocity loads on backend server

### Steps to reproduce

add plugin to plugins folder

### VelocityVanish's configuration file

```# Full wiki is available at https://github.com/Syrent/VelocityVanish/wiki
config_version: 7
default_language: en_US
show_dependency_suggestions: true

# IMPORTANT: Legacy code support is strongly discouraged. It adversely affects performance and lacks new Minecraft chat features.
support_legacy_color_codes: false

# If you're the first player to join the server, Vanish data may not synchronize correctly,
# and you may not enter vanish mode. With this option enabled, you can still enter vanish mode
# if you have operator privileges or the "velocityvanish.action.vanish.force" permission.
# If you weren't previously in vanish mode, you'll be unvanished after a few seconds.
force_vanish_if_first: true

bstats: true

# Leave sounds blank to disable them
# Example: command: ""
sounds:
  command: ENTITY_EXPERIENCE_ORB_PICKUP
  vanish: ENTITY_EXPERIENCE_ORB_PICKUP
  unvanish: ENTITY_EXPERIENCE_ORB_PICKUP

vanish:
  # Show players an actionbar when they are vanished
  actionbar: true
  
  # Remember vanish data after switching servers?
  # Note: Requires Velocity as a proxy server
  remember: true
  
  # Should we change player type to spectator in tablist for players who have permission to see vanished players?
  # Note: Requires ProtocolLib in versions lower than 1.19
  see_as_spectator: true
  
  # Should vanished players be protected from all damage sources?
  invincible: true
  
  # You can also achieve this by setting the fake join/leave message to blank in the language file
  fake_join_leave_message: true
  
  silent_open_container: true
  
  prevent:
    pickup: true
    block_break: false
    block_place: false
    interact: false
    # Only works on PaperMC servers (Paper, Purpur, Pufferfish, etc...)
    advancement: true

hooks:
  essentials:
    force_prevent_private_messages: false
```
### Plugins

[19:20:12 INFO]: Paper Plugins: (1):
[19:20:12 INFO]:  - VelocityVanish
[19:20:12 INFO]: Bukkit Plugins: (56):
[19:20:12 INFO]:  - ajQueue, AvNCombat, AvnEnchants, AvNItems, AvNMusic, AvNRep, AvnTeleport, AvNTitles, BossShopPro, Citizens
[19:20:12 INFO]:  CMI, CMILib, CoreProtect, DamageIndicator, Denizen, DungeonParties, F3NPerm, FastAsyncVoxelSniper, FastAsyncWorldEdit, FreedomChat
[19:20:12 INFO]:  GlowAPI, HappyHUD, HeadDatabase, Lands, LibsDisguises, LuckPerms, MapModCompanion, MythicCrucible, MythicDungeons, MythicMobs
[19:20:12 INFO]:  PacketListenerApi, PAPIProxyBridge, PlaceholderAPI, PlasmoVoice, PlugManX, ProtocolLib, pv-addon-discs, pv-addon-lavaplayer-lib, pv-addon-priority, pv-addon-sculk
[19:20:12 INFO]:  pv-addon-spectator, pv-addon-whisper, QuickShop, spark, squaremap, squaremap-banners, squaremap-signs, squaremap-vanish, squaremap-worldguard, SyncCommands
[19:20:12 INFO]:  TAB-Bridge, UnifiedMetrics, Vault, VentureChat, ViaVersion, WorldGuard

### Additional info

[2023-09-20-9.log](https://github.com/Syrent/VelocityVanish/files/12676826/2023-09-20-9.log)

### Checklist

- [X] I am running the latest version of the plugin on both Velocity and Spigot
- [X] I have read the wiki to make sure it's not an issue with configuration
- [X] I tried the plugin on a fresh server and it didn't work (Or it worked but I know exactly which plugin it's incompatible with)
- [ ] I ticked all of the boxes without actually reading them

**Comment by Frerduro** [comment]
[latest (2).log](https://github.com/Syrent/VelocityVanish/files/12676898/latest.2.log)
VelocityVanish+Protocollib

**Comment by Syrent** [comment]
I'm unable to reproduce the issue on my server. You can check the [latest.log](https://github.com/Syrent/VelocityVanish/files/12678285/latest.log) for more details. It's possible that there is an issue with your ProtocolLib jar file.

**Comment by Frerduro** [comment]
5.1.0 protocollib + Vanish https://gist.github.com/Frerduro/21ff4711bce258043c76e2f94c91c4be

**Comment by Syrent** [comment]
Workaround implemented in VelocityVanish 3.26.0 build 26.
The problem is Purpur 1.19.4 doesn't use the latest Paper modern plugin strategy version.
